depends/packages/boost.mk

```diff
@@ -27,7 +27,7 @@ else
 $(package)_toolset_$(host_os)=gcc
 endif
 $(package)_config_libraries=filesystem,test
-$(package)_cxxflags=-std=c++17
+$(package)_cxxflags=-std=c++11
```

Comment:
could you add TODO here so far as it is not permanent change?

Reply:
Mentioned it in https://github.com/dashpay/dash-issues/issues/53

Reply:
I meant TODO right here, so, next release we can re-evaluate it if it is still necessary or can be removed, but I guess you should not forget it yourself :D